src/Microsoft.Health.Fhir.Core/Microsoft.Health.Fhir.Core.csproj

```diff
@@ -29,6 +29,16 @@
     <EmbeddedResource Include="Data\Stu3\unsupported-search-parameters.json" />
     <EmbeddedResource Include="Data\Stu3\resourcepath-codesystem-mappings.json" />
     <EmbeddedResource Include="Features\Security\roles.schema.json" />
+    <None Remove="Data\R4B\BaseCapabilities.json" />
```

Comment:
Are the None lines needed here? They aren't present for any of the other embedded resources.